apps/gauzy/src/app/pages/dashboard/project-management/project-management-details/project-management-details.component.scss

```
@@ -53,11 +53,14 @@ $gap: 1rem;
       width: 100%;
       cursor: pointer;
       .dot {
-        border: 1px solid #6e49e8;
+        // Was a literal `#6e49e8` on both the ring and the checked fill — the
```

Stylelint (17.14.1)

[error] Unknown word Was (CssSyntaxError)

(CssSyntaxError)